src/foam/nanos/notification/NotificationView.js

```diff
@@ -8,43 +8,124 @@ foam.CLASS({
   package: 'foam.nanos.notification',
   name: 'NotificationView',
   extends: 'foam.u2.View',
+  documentation: 'Notification controller',
 
-  axioms: [
-    foam.pattern.Faceted.create()
```

Comment:
@kgrgreer can you comment on this.
Believe was important so a Notification view could be replaced with a custom view.

Reply:
NotificationView was renamed to NotificationCitationView

Reply:
see NotificationCitationView